File: Python/tests/test_task2AFC_sdt.py
import pytest

from hbayesdm.models import task2AFC_sdt


def test_task2AFC_sdt():
    _ = task2AFC_sdt(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_task2AFC_sdt.R
context("Test task2AFC_sdt")
library(hBayesDM)

test_that("Test task2AFC_sdt", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(task2AFC_sdt(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
